lib/Munin/Master/Update.pm

```diff
@@ -275,7 +275,7 @@ sub _db_init {
 	$dbh->do("CREATE UNIQUE INDEX IF NOT EXISTS pk_state ON state (type, id)");
 
 	# Munin stats
-	$dbh->do("CREATE TABLE IF NOT EXISTS stats (runid VARCHAR NOT NULL, tstp DATETIME, type VARCHAR, name VARCHAR, duration NUMERIC)");
+	$dbh->do("CREATE TABLE IF NOT EXISTS stats (runid VARCHAR NOT NULL, tstp TIMESTAMP, type VARCHAR, name VARCHAR, duration NUMERIC)");
```

Comment:
I'm not sure that I'd like to have timezones in the DB. Why do you think it would be a good idea?

Reply:
Right now it is an undefined time, or?

Reply:
The type that CURRENT_TIMESTAMP yields is TIMESTAMP WITH TIME ZONE. So if it lands into a timestamp without time zone column, you're doing extra work converting one type into another. The point is though that you want to use timestamptz unless you can argue otherwise.